src/resources/language/_language-zh.yml

```diff
@@ -0,0 +1,62 @@
+toc-title-document: "目录"
+toc-title-website: "在该页面"
+
+section-title-footnotes: "脚注"
+
+callout-tip-caption: "提示"
+callout-note-caption: "注释"
+callout-warning-caption: "警告"
+callout-important-caption: "重要"
+callout-danger-caption: "危险"
+
+code-summary: "代码"
+
+code-tools-menu-caption: "代码"
+code-tools-show-all-code: "显示所有代码"
+code-tools-hide-all-code: "隐藏所有代码"
+code-tools-view-source: "查看代码"
+code-tools-source-code: "源代码"
+
+copy-button-tooltip: "复制到剪贴板"
+
+repo-action-links-edit: "编辑该页面"
+repo-action-links-source: "查看代码"
+repo-action-links-issue: "报告问题"
+
+search-no-results-text: "没有结果"
+search-matching-documents-text: "匹配的文档"
+search-copy-link-title: "复制搜索链接"
+search-hide-matches-text: "隐藏其它匹配结果"
+search-more-match-text: "更多匹配结果"
+search-more-matches-text: "更多匹配结果"
+search-clear-button-title: "清除"
+search-detached-cancel-button-title: "取消"
+search-submit-button-title: "提交"
+
+crossref-fig-title: "图"
+crossref-tbl-title: "表格"
+crossref-lst-title: "列表"
+crossref-thm-title: "定理"
+crossref-lem-title: "引理"
+crossref-cor-title: "推论"
+crossref-prp-title: "命题"
+crossref-cnj-title: "猜想"
+crossref-def-title: "定义"
+crossref-exm-title: "例子"
+crossref-exr-title: "练习"
+crossref-fig-prefix: "图."
+crossref-tbl-prefix: "表."
+crossref-lst-prefix: "列表."
+crossref-sec-prefix: "节."
+crossref-eq-prefix: "等式."
+crossref-thm-prefix: "定理."
+crossref-lem-prefix: "引理."
+crossref-cor-prefix: "推论."
+crossref-prp-prefix: "命题."
+crossref-cnj-prefix: "猜想."
+crossref-def-prefix: "定义."
+crossref-exm-prefix: "例."
+crossref-exr-prefix: "练习."
+crossref-lof-title: "插图目录"
+crossref-lot-title: "列表目录"
+crossref-lol-title: "列表目录"
```

Comment:
Meanwhile the good news is all of those are pretty unambiguous :slightly_smiling_face: